recipes/naive-tsearch/all/conanfile.py

```diff
@@ -1,16 +1,19 @@
-from conans import CMake, ConanFile, tools
+from conan import ConanFile
+from conan.tools.files import get, copy, rm, rmdir
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
 import os
 
+required_conan_version = ">=1.53.0"
 
 class NaiveTsearchConan(ConanFile):
     name = "naive-tsearch"
     description = "A simple tsearch() implementation for platforms without one"
-    topics = ("conan", "naive-tsearch", "tsearch", "search", "tree", "msvc")
     license = "MIT"
-    homepage = "https://github.com/kulp/naive-tsearch"
     url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kulp/naive-tsearch"
+    topics = ("tsearch", "search", "tree", "msvc")
+    package_type = "static-library"
```

Comment:
```suggestion
```

This library can be header-only and it will lie about the package type. As this package can be static library or header-only, and the package is different between them, the best choice is removing the package type and giving the responsibility to Conan client to detect the package type based on the generated artifacts.

Reply:
@uilianries 
I see this policy.
Thank you for your comments!